src/VarUnit.h

```diff
@@ -76,7 +76,7 @@ class VarUnit
 
 private:
     TVar* base;
-    QSet<QString> varList;
+    QSet<QString> variableList;
```

Comment:
Well that is awkward - I wouldn't have guessed from the name that it is a `QSet<T>`.

Just an observation for this PR, you might wish to action it but I guess that'll be a separate PR.

Reply:
Applied.